src/time_zone_format.cc

```diff
@@ -18,6 +18,15 @@
 # endif
 #endif
 
+#if HAS_STRPTIME
+# if !defined(_XOPEN_SOURCE)
+#  define _XOPEN_SOURCE  // Definedness suffices for strptime.
+# endif
+// Include time.h directly since, by C++ standards, ctime doesn't have to
```

Comment:
Should we move this `#include` part down to the `HAS_STRPTIME` section in the includes below?

Reply:
Done. The defines have to stay above the first #include though, since it transitively #include's chrono, which can include ctime, which can include time.h :|